test/specs/visualforceLsp.e2e.ts

```diff
@@ -0,0 +1,137 @@
+      'Developer: Show Running Extensions',
+      1
+    );
+
+    // Close panel so the visualforce extension can be seen
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'View: Close Panel',
+      1
+    );
+
+    // Verify Visualforce extension is present and running
+    const extensionNameDivs = await $$('div.name');
+    let extensionWasFound = false;
+    for (const extensionNameDiv of extensionNameDivs) {
+      const text = await extensionNameDiv.getText();
+      if (text.includes('salesforce.salesforcedx-vscode-visualforce')) {
+        extensionWasFound = true;
+      }
+    }
+    expect(extensionWasFound).toBe(true);
+  });
+
+  step('Go to Definition', async () => {
+    utilities.log(`${testSetup.testSuiteSuffixName} - Go to Definition`);
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = (await editorView.openEditor(
+      'FooPage.page'
+    )) as TextEditor;
+    await textEditor.moveCursor(1, 25);
+
+    // Go to definition through F12
+    await browser.keys(['F12']);
+    await utilities.pause(1);
+
+    // TODO: go to definition is actually not working
+
+    // // Verify 'Go to definition' took us to the definition file
+    // const activeTab = await editorView.getActiveTab();
+    // const title = await activeTab?.getTitle();
+    // expect(title).toBe('MyController.cls');
+  });
+
+  step('Autocompletion', async () => {
+    utilities.log(`${testSetup.testSuiteSuffixName} - Autocompletion`);
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = (await editorView.openEditor(
+      'FooPage.page'
+    )) as TextEditor;
+    await textEditor.typeTextAt(3, 1, '\t\t<apex:pageM');
+    await utilities.pause(1);
+
+    // Verify autocompletion options are present
+    const autocompletionOptions = await $$(
+      'textarea.inputarea.monaco-mouse-cursor-text'
+    );
+    const ariaHasPopupAttribute = await autocompletionOptions[0].getAttribute(
+      'aria-haspopup'
+    );
+    expect(ariaHasPopupAttribute).toBe('true');
+
+    const ariaAutocompleteAttribute = await autocompletionOptions[0].getAttribute(
+      'aria-autocomplete'
+    );
+    expect(ariaAutocompleteAttribute).toBe('list');
+
+    // Verify autocompletion options can be selected and therefore automatically inserted into the file
+    await browser.keys(['Enter']);
+    await textEditor.typeText('/>');
+    await textEditor.save();
+    await utilities.pause(1);
+    const line3Text = await textEditor.getTextAtLine(3);
+    expect(line3Text).toContain('apex:pageMessage');
+  });
+
+  step('Tear down and clean up the testing environment', async () => {
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - Tear down and clean up the testing environment`
+    );
+    await testSetup.tearDown();
+  });
+});
```

Comment:
It looks like the "SFDX: Create Visualforce Page" command is covered, but the "SFDX: Create Visualforce Component" command isa lacking coverage.

Reply:
Yup, that should be covered in the templates suite 
<img width="1337" alt="image" src="https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/113132642/fb4aa2b3-f0e9-4332-b397-cca21d49a811">

Reply:
Ah, got it, thanks.  At the time (this morning) I was thinking it was to be here in this suite.